utils/sql/git/required/2022_04_30_character_peqzone_flags.sql

```diff
@@ -0,0 +1,14 @@
+SET NAMES utf8mb4;
+SET FOREIGN_KEY_CHECKS = 0;
+
+-- ----------------------------
+-- Table structure for character_peqzone_flags
+-- ----------------------------
+DROP TABLE IF EXISTS `character_peqzone_flags`;
+CREATE TABLE `character_peqzone_flags`  (
+  `character_id` int NOT NULL DEFAULT 0,
```

Comment:
Would prefer if we used `id` here. I know our other tables are all over the place